src/OrchardCore.Modules/Orchard.Themes/Orchard.Themes.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\OrchardCore\Microsoft.AspNetCore.Modules.Abstractions\Microsoft.AspNetCore.Modules.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.AsModule\OrchardCore.AsModule.csproj" />
```

Comment:
Sorry, I just searched for "Theme" to check, and got this one as the first result, didn't look at the name ;)